ReactCommon/react/renderer/attributedstring/TextAttributes.cpp

```diff
@@ -97,6 +97,10 @@ void TextAttributes::apply(TextAttributes textAttributes) {
   accessibilityRole = textAttributes.accessibilityRole.hasValue()
       ? textAttributes.accessibilityRole
       : accessibilityRole;
+
+  // Vertical Alignment
+  textAlignVertical = !textAttributes.textAlignVertical.empty() ? textAttributes.textAlignVertical
```

Comment:
why is ! required ? we can change the condition itself ?

Reply:
To keep similar to the current implementation